addon/templates/components/validated-input/types/-themes/bootstrap/checkbox-group.hbs

```diff
@@ -0,0 +1,14 @@
+{{#each options as |option i|}}
+  <div class="custom-control custom-checkbox">
+    <input
+      type     = "checkbox"
```

Comment:
Is prettier not the default for template in this addon or is this actually prettier?

Reply:
It seems that prettier is not enabled on `.hbs` files in this repo. The formatting is, again, inspired by similar other files.